fulp_modules/main_features/jobs/deputy/code/deputy_clothing.dm

```diff
@@ -4,56 +4,44 @@
 
 	id_trim = /datum/id_trim/job/deputy
 	uniform = /obj/item/clothing/under/rank/security/officer/mallcop
-	ears = null
-	belt = /obj/item/pda/security
+	ears = /obj/item/radio/headset/headset_sec/alt
+	belt = /obj/item/storage/belt/security/deputy
 	glasses = /obj/item/clothing/glasses/hud/security/sunglasses
 	gloves = /obj/item/clothing/gloves/color/black
 	shoes = /obj/item/clothing/shoes/laceup
 	l_pocket = /obj/item/flashlight/seclite
-	r_pocket = /obj/item/assembly/flash/handheld
+	r_pocket = /obj/item/pda/security
 	backpack = /obj/item/storage/backpack/security
 	satchel = /obj/item/storage/backpack/satchel/sec
 	duffelbag = /obj/item/storage/backpack/duffelbag/sec
 
+	backpack_contents = list(
+		/obj/item/melee/baton/loaded = 1,
```

Comment:
I'd suggest moving the holosign creator here and putting the baton in their secbelt.

... I'd also suggest just giving them a normal secbelt instead of giving them one that has a near-useless (in 99% of circumstances) smoke grenade instead of a flashbang, but that's just me.

Reply:
also, giving deputies a spare set of handcuffs or two would also probably help suppress the urge to go to sec.

Reply:
I had it in the bag instead because security officers had it there, but I'll swap it.